### ISSUE TITLE
Stricter rules for delegates's profile pictures

### ISSUE THREAD

**Issue by jfly** [thewca/worldcubeassociation.org#2674]
The WQAC would like to enforce stricter rules for WCA Delegates's profile pictures. We have a few things to do:

1. Update the guidelines on https://www.worldcubeassociation.org/profile/edit?section=avatar for delegates. We're still discussing the exact wording, but they should look something like this:

> - The avatar must be of your current self, depicted in the same fashion as how you would appear at a competition as the WCA Delegate.
> - The avatar must be framed, cropped, angled, and lit, so that any competitor or spectator at a competition can use it to unambiguously identify you at a competition venue.
> - The thumbnail of the avatar is only a section of the avatar, and it will be used to represent your full avatar. The thumbnail must be adjusted, so you can be recognized from it alone, and it must contain your face.

2. Update the guidelines on https://www.worldcubeassociation.org/admin/avatars with a version of the above bullet points, but written from the perspective of reminding the WRT that delegates should be handled specially.

3. Update https://www.worldcubeassociation.org/admin/avatars to include a preview of the users thumbnail.

4. Update https://www.worldcubeassociation.org/admin/avatars to make it visually clear when a user is a delegate, so the WRT knows to apply these new special delegate-only rules to them.

5. Update the list of things to do when promoting someone to a candidate delegate to include verifying their profile picture is up to delegate-snuff. @pedrosino is there a doc somewhere with a list of things to do when promoting someone to be a candidate delegate?

**Open question** Are delegates now required to have a profile picture? If yes, does that affect any of the above?

**Comment by pedrosino** [comment]
Yes, we have a spreadsheet with the steps. I'll add this there.

For 6, I expect the WQAC will do it.

**Comment by AlbertoPdRF** [comment]
@Luis-J-Ianez already did 6. not so long ago, although some of the newest Delegates may have avatars not compliant with the guidelines.

Regarding your open question: right now there are Delegates without profile picture, and I guess it's up to the WQAC to decide if that's acceptable or not.

**Comment by DanielEgdal** [comment]
Hi, is there any update on this topic? I believe we have covered most of your concers on the mailing thread. If you have more concers, please ask them, and we will try to answer them :)

**Comment by DanielEgdal** [comment]
Hi, the WQAC is still waiting for an update. Again, if you have any further concerns, which you didn't feel got adressed in the mailing thread, then please feel free to ask.

**Comment by DanielEgdal** [comment]
@pedrosino Yes! Then only display these to Delegates, in addition to the 'old' guidelines.
